examples/api/test/widgets/nested_scroll_view/nested_scroll_view.0_test.dart

```diff
@@ -41,4 +41,46 @@ void main() {
       lessThan(initialAppBarHeight),
     );
   });
+
+  testWidgets('Maintains scroll position of inactive tab', (WidgetTester tester) async {
```

Comment:
I'm wondering why there is a test in examples and not one in lib/src? Imagining an unlikely future where someone throws out examples and the associated tests (because who cares about meagre examples anyways), couldn't this change be regressed?

Reply:
I added the test here so that I could use `NestedScrollViewExampleApp()` instead of configuring the huge widget tree.

But this is a good point—and I think it'd be easy to just import the example to the lib/src test file. I'll go ahead and do that, thanks!

Reply:
…well dang, it looks like importing an example into a flutter/test library currently isn't possible.

Moving the test wouldn't guarantee a lack of regressions; for example, https://github.com/flutter/flutter/pull/50831 changed a test's expected output, and then https://github.com/flutter/flutter/pull/149819 attempted to change it back.

Overall, I'm not sure exactly how relocating the test would affect the probability of regressions. It's not something I'm super worried about, though I'd be open to hearing other perspectives.

Reply:
Isn't there a way to reproduce the bug in flutter/test directly? i.e strip the example down to the essentials causing the bug.

Reply:
Yes, though it wouldn't be stripped down by much.

Reply:
My suggestion in that case would be to configure the large widget tree in flutter/test, unfortunately. Just so that in the event that examples/ and packages/flutter are someday separated, the change isn't untested in packages/flutter. If that's not possible I wonder if there are any precedents, would also like to hear what others think.

Reply:
Although I haven't looked into this example too deeply so maybe I am missing some important context.

Reply:
> My suggestion in that case would be to configure the large widget tree in flutter/test, unfortunately. Just so that in the event that examples/ and packages/flutter are someday separated, the change isn't untested in packages/flutter.

Sounds good.

I really appreciate the feedback, have a great Halloween! 👻